app/services/proxy_status/tftp.rb

```diff
@@ -0,0 +1,24 @@
+module ProxyStatus
+  class TFTP < Base
+    def initialize(proxy, opts = {})
+      super(proxy, opts, 'TFTP')
+    end
+
+    def tftp_server
+      Rails.cache.fetch("proxy_#{proxy.id}/tftp_server", :expires_in => cache_duration) do
```

Comment:
can we extend the fetch_proxy_data to accept opts and then you can define if you want to cache or not? this pattern is going to not DRY alot